src/node/internal/internal_http_server.ts

```diff
@@ -0,0 +1,708 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+
+import {
+  ERR_METHOD_NOT_IMPLEMENTED,
+  ERR_HTTP_HEADERS_SENT,
+  ERR_HTTP_INVALID_STATUS_CODE,
+  ERR_INVALID_CHAR,
+  ERR_INVALID_ARG_VALUE,
+  ERR_OUT_OF_RANGE,
+  ERR_OPTION_NOT_IMPLEMENTED,
+  ERR_INVALID_ARG_TYPE,
+  ERR_SERVER_ALREADY_LISTEN,
+} from 'node-internal:internal_errors';
+import { EventEmitter } from 'node-internal:events';
+import {
+  kUniqueHeaders,
+  OutgoingMessage,
+  parseUniqueHeadersOption,
+} from 'node-internal:internal_http_outgoing';
+import {
+  validateBoolean,
+  validateInteger,
+  validateObject,
+  validatePort,
+} from 'node-internal:validators';
+import { portMapper } from 'cloudflare-internal:http';
+import { IncomingMessage } from 'node-internal:internal_http_incoming';
+import {
+  kOutHeaders,
+  WrittenDataBufferEntry,
+  HeadersSentEvent,
+} from 'node-internal:internal_http_outgoing';
+import {
+  chunkExpression,
+  _checkInvalidHeaderChar,
+} from 'node-internal:internal_http';
+import { _normalizeArgs } from 'node-internal:internal_net';
+import { Buffer } from 'node-internal:internal_buffer';
+
+import type {
+  Server as _Server,
+  ServerResponse as _ServerResponse,
+  RequestListener,
+  ServerOptions,
+  OutgoingHttpHeaders,
+  OutgoingHttpHeader,
+} from 'node:http';
+import type { Socket } from 'node:net';
+
+export const kServerResponse = Symbol('ServerResponse');
+export const kConnectionsCheckingInterval = Symbol(
+  'http.server.connectionsCheckingInterval'
+);
+
+export const STATUS_CODES = {
```

Comment:
Is this different from the `STATUS_CODES` in `src/node/internal/internal_http_constants.ts` ?

Reply:
It's the same. You're right.